scripts/eurostat/regional_statistics_by_nuts/population_density/PopulationDensity_preprocess_gen_tmcf.py

```diff
@@ -16,22 +16,19 @@
 import io
 import csv
 
-# data_url = "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/BulkDownloadListing?file=data/demo_r_d3area.tsv.gz"
-
-_SOURCE_TSV = "./demo_r_d3dens.tsv"
-_SOURCE_CSV_LONG = "./demo_r_d3dens.csv"
+_DATA_URL = "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/BulkDownloadListing?file=data/demo_r_d3dens.tsv.gz"
 _CLEANED_CSV = "./PopulationDensity_Eurostat_NUTS3.csv"
 _TMCF = "./PopulationDensity_Eurostat_NUTS3.tmcf"
 
-_OUTPUT_COLUMNS = [
```

Comment:
We should leave it in all caps and with the initial underscore.  The latter makes it private to this file.  The former is what the style guide suggests for global variables.  Q.v., https://google.github.io/styleguide/pyguide.html#316-naming.